src/java.desktop/macosx/classes/sun/lwawt/macosx/CPlatformWindow.java

```diff
@@ -124,6 +127,10 @@ private static native void nativeSetNSWindowStandardFrame(long nsWindowPtr,
     public static final String WINDOW_TRANSPARENT_TITLE_BAR = "apple.awt.transparentTitleBar";
     public static final String WINDOW_TITLE_VISIBLE = "apple.awt.windowTitleVisible";
 
+    // This system property is used in javafx as well
```

Comment:
Reword this comment to mentioned that it is named as jdk.* because the property is not specific to AWT and it will be used in JavaFX also.

Reply:
yeah , instead of "will be used" we should used "is used" because that will be situation after some time.